Jenkinsfile

```diff
@@ -444,8 +445,137 @@ pipeline {
+                                             aggregatingResults: true,
+                                             id: "analysis-sles12.3",
+                                             tools: [ gcc4(), cppCheck() ],
+                                             filters: [excludeFile('.*\\/_build\\.external-Linux\\/.*'),
+                                                       excludeFile('_build\\.external-Linux\\/.*')]
+                                */
+                            }
+                            /* when JENKINS-39203 is resolved, can probably use stepResult
+                               here and remove the remaining post conditions
+                               stepResult name: env.STAGE_NAME,
+                                          context: 'build/' + env.STAGE_NAME,
+                                          result: ${currentBuild.currentResult}
+                            */
+                        }
+                        success {
+                        /* temporarily moved into stepResult due to JENKINS-39203
+                            githubNotify credentialsId: 'daos-jenkins-commit-status',
+                                         description: env.STAGE_NAME,
+                                         context: 'build/' + env.STAGE_NAME,
+                                         status: 'SUCCESS'
+                            */
+                            sh "rm -rf _build.external${arch}"
+                        }
+                        unstable {
+                            sh "mv config${arch}.log config.log-sles12sp3-gcc"
+                            archiveArtifacts artifacts: 'config.log-sles12sp3-gcc'
+                            /* temporarily moved into stepResult due to JENKINS-39203
+                            githubNotify credentialsId: 'daos-jenkins-commit-status',
+                                         description: env.STAGE_NAME,
+                                         context: 'build/' + env.STAGE_NAME,
+                                         status: 'FAILURE'
+                            */
+                        }
+                        failure {
+                            sh "mv config${arch}.log config.log-sles12sp3-gcc"
+                            archiveArtifacts artifacts: 'config.log-sles12sp3-gcc'
+                            /* temporarily moved into stepResult due to JENKINS-39203
+                            githubNotify credentialsId: 'daos-jenkins-commit-status',
+                                         description: env.STAGE_NAME,
+                                         context: 'build/' + env.STAGE_NAME,
+                                         status: 'ERROR'
+                            */
+                        }
+                    }
+                }
+                stage('Build on LEAP 42.3') {
+                    when { beforeAgent true
+                           environment name: 'LEAP42_3_DOCKER', value: 'true' }
```

Comment:
Don't we need to enable this global variable in the production Jenkins before landing this so that these new builds run successfully?

Reply:
Yes to be active.  Needs a chef recipe change make it permanent.
Generally a site with a SLES license would not need to enable both SLES and LEAP builds.

Reply:
Right, so the question remains if we are going to make this (chef, and production Jenkins) change to set the required global variables before we land this?

Unfortunately GitHub doesn't have any way to block landing a PR on another issue/PR being open/closed, which is what I'd like to see here.  Unless I am misunderstanding something about the order of doing things here.